nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RITSupport.java

```diff
@@ -119,29 +80,99 @@
 
   public static final String CONTENT_TYPE_X_XZ = "application/x-xz";
 
-  public static final String PACKAGES_GZ_FILE_NAME = format("%s%s", PACKAGES_NAME, GZ_EXT);
-
-  public static final String PACKAGES_RDS_FILE_NAME = format("%s%s", PACKAGES_NAME, RDS_EXT);
-
-  public static final String ARCHIVE_RDS_FILE_NAME = format("%s%s", ARCHIVE_RDS_NAME, RDS_EXT);
-
-  public static final String AGRICOLAE_PATH_FULL_131_TGZ =
-      String.format("%s/%s", PKG_GZ_PATH, AGRICOLAE_PKG_FILE_NAME_131_TGZ);
-
-  public static final String AGRICOLAE_PATH_FULL_131_TARGZ =
-      String.format("%s/%s", PKG_GZ_PATH, AGRICOLAE_PKG_FILE_NAME_131_TARGZ);
-
-  public static final String AGRICOLAE_PATH_FULL_121_TARGZ =
-      String.format("%s/%s", PKG_GZ_PATH, AGRICOLAE_PKG_FILE_NAME_121_TARGZ);
-
-  public static final String AGRICOLAE_PATH_FULL_WRONG_EXTENSION_XXX =
-      String.format("%s/%s", PKG_GZ_PATH, AGRICOLAE_PKG_FILE_NAME_WRONG_EXTENSION_XXX);
-
-  public static final String PACKAGES_GZ_PATH_FULL = format("%s/%s", PKG_GZ_PATH, PACKAGES_GZ_FILE_NAME);
+  public static final String RDS_METADATA_KIND = "RDS_METADATA";
 
-  public static final String PACKAGES_RDS_PATH_FULL = format("%s/%s", PKG_RDS_PATH, PACKAGES_RDS_FILE_NAME);
+  public static final String PACKAGES_KIND = "PACKAGES";
 
-  public static final String ARCHIVE_RDS_PATH_FULL = format("%s/%s", ARCHIVE_RDS_PATH, ARCHIVE_RDS_FILE_NAME);
+  public static final TestPackage AGRICOLAE_131_TGZ = new TestPackage(
+      "agricolae_1.3-1.tgz",
+      PKG_BIN_PATH,
+      "agricolae",
+      "1.3-1",
+      ".tgz",
+      CONTENT_TYPE_X_TGZ
+  );
+
+  public static final TestPackage AGRICOLAE_131_TARGZ = new TestPackage(
+      "agricolae_1.3-1.tar.gz",
+      PKG_SRC_PATH,
+      "agricolae",
+      "1.3-1",
+      ".tar.gz",
+      CONTENT_TYPE_X_GZIP
+  );
+
+  public static final TestPackage AGRICOLAE_121_TARGZ = new TestPackage(
+      "agricolae_1.2-1.tar.gz",
+      PKG_SRC_PATH,
+      "agricolae",
+      "1.2-1",
+      ".tar.gz",
+      CONTENT_TYPE_X_GZIP
+  );
+
+  public static final TestPackage AGRICOLAE_101_TARGZ = new TestPackage(
+      "agricolae_1.0-1.tar.gz",
+      PKG_SRC_PATH,
+      "agricolae",
+      "1.0-1",
+      ".tar.gz",
```

Comment:
looks like better to use tar.gz like constants

Reply:
done in 9f5f258e702072bd3da6485af279ce00607c68db